api/lib/gateways/Jigsaw/FetchNotes.js

```diff
@@ -49,12 +50,47 @@ module.exports = options => {
 
     return await doGetRequest(collabCaseworkMessagesUrl(smsContact[0].id));
   };
+
+  const processSMS = sms => {
+    return sms.map(m => {
+      return buildNote({
+        title: `${m.outgoing ? 'Outgoing' : 'Incoming'} SMS`,
+        text: m.message,
+        date: m.time,
+        user: m.username,
+        system: 'SMS'
+      });
+    });
+  };
+
+  const processNotes = (notes, noteType) => {
+    return notes.map(note => {
+      const date = note.interviewDate ? note.interviewDate : note.createdDate;
+      return buildNote({
+        title: noteType,
+        text: note.content,
+        date: date,
+        user: note.officerName,
+        system: Systems.JIGSAW
+      });
+    });
+  };
   return {
     execute: async (id, token) => {
+      try {
       const jigsaw_id = await fetchSystemId(id);
```

Comment:
why are we using fetchSystemId here? check what we are doing in the old jigsaw notes backend

Reply:
In query handler when calling backends we would call them with remote_id, so I thought every note gateway should fetch that id? 
<img width="656" alt="image" src="https://user-images.githubusercontent.com/54268893/74229613-a8551e00-4cba-11ea-8fe5-5846c388c774.png">

Reply:
cool